docs/setup.md

```diff
@@ -62,17 +62,24 @@ You can also contact us on slack for any further queries.
 ## TL;DR bare minimum version
 If you know your way around Rails, here's the very short version. Some additional requirements are necessary to make all the tests pass and all the features work, but this should be enough to stand up the app quickly.
 
-* copy `config/application.example.yml` to `config/application.yml`
-* copy `config/database.example.yml` to `config/database.yml`
-* create a MySQL database, `dashboard`
-* install ruby 2.7.1 and nodejs
-* `bundle install`
-* `rake db:migrate`
-* install yarn
-* `yarn` for more javascript requirements
-* `yarn start` to build assets
-* `guard` or `rails s` to start a server
-* localhost:3000 should load the home page
+1. Fork our repo from https://github.com/WikiEducationFoundation/WikiEduDashboard.
+2. In the console, download a copy of your forked repo with `git clone https://github.com/your_username/WikiEduDashboard.git` where `your_username` is your GitHub     username.
+3. Enter the new WikiEduDashboard directory with `cd WikiEduDashboard`.
+4. **Install gems:** Run `bundle install`
+ <br />**Note:** If you get an error message like Your Ruby version is 2.x.x, then you need to install the ruby version 2.7.1 using rvm or rbenv.
```

Comment:
Rather than explicitly include the ruby version, it might be better to note that the required ruby version is specified in `Gemfile`. The version changes periodically, and the fewer places in the documentation that need to be updated each time, the better.

Reply:
It's not completely necessary to use either rvm or rbenv; any method of installing a specific Ruby version should be fine. Maybe "... then you need to install the required Ruby version (eg, using rvm or rbenv)."

Reply:
Yaa you're right that there are multiple ways to download the Ruby version or to copy files to the required location but I feel it gets easy/convenient for a user to copy and paste commands directly into the terminal rather than googling it, moving files, or generating unnecessary errors. What's your opinion on this?